js/blob.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import { Blob, BlobPart, BlobPropertyBag } from "./fetch_types";
+import { containsOnlyASCII } from "./util";
+
+const bytesSymbol = Symbol("bytes");
+
+export class DenoBlob implements Blob {
```

Comment:
BlobImpl I think is better ?

Reply:
@ry I followed the same semantics as `DenoHeaders`, but let me know if you want to change it : )

Reply:
Ya... sorry I’m waffling on the naming. Let’s keep it as is now - maybe we can rename them all using “impl” at once.